cache/image_cache.go

```diff
@@ -58,7 +58,7 @@ func NewImageCacheFromName(name string, keychain authn.Keychain, logger log.Logg
 }
 
 func (c *ImageCache) Exists() bool {
-	return c.origImage.Found()
+	return c.origImage.Valid()
```

Comment:
Here we are delegating `Exists()` to `Valid` but in `analyzer.go` we are explicitly doing both `.Found()` and `.Valid()`. Is that because we want specific Not Found/Corrupt messages in analzyer but it doesn't much matter for cache images?

Reply:
I just wanted to preserve the log message that was already present in `getImageIdentifier`. The `.Valid()` function returns false if the image doesn't exist, so `i.Exists() && i.Valid()` will send one redundant `HEAD` request to confirm that manifest exits.

Reply:
Found a better way to do this that I think makes more sense and provides better logging.